src/test/java/calculator/CalculatorTest.java

```diff
@@ -0,0 +1,66 @@
+package calculator;
+
+import org.junit.jupiter.api.Test;
+import static org.assertj.core.api.Assertions.*;
+
+public class CalculatorTest {
+
+    Calculator calculator = new Calculator();
+    
+    @Test
+    public void splitAndSum_null_또는_빈문자() throws Exception {
+        assertThat(calculator.calculate
+                (calculator.stringArrToIntArr
+                        (calculator.splitText(null))
+                )
+        ).isEqualTo(0);
```

Comment:
저의 의도와는 다르게 전달된 것 같네요 🥲
```suggestion
        assertThat(calculator.calculate(null)).isEqualTo(0);
```

경범님이 제공하는 방식은 절차지향적인 방식으로 접근하신 것 같아요.
우리는 객체지향을 배우고 있으니 Calculator 객체에게 계산을 요청하는 방향으로 접근해야 합니다.

하나의 예를들어 경범님이 친구에게 '연필 줘' 라는 요청을 했을 때
친구가 가방 또는 주머니에서 **알아서 연필을 꺼내** 경범님에게 줄 것으로 예상합니다.

다른 방법으로 경범님이 친구에게 '친구야 가방열어', 그리고 '가방에서 연필찾아', '연필 꺼내', '가방 닫고 줘'
이렇게 한다면 친구를 존중하는 행위일까요? 저는 그렇지 않다고 봐요

조영호님의 오브젝트라는 책을 읽어보면 객체를 존중해야한다고 하는데요
마찬가지 Calculator 객체를 존중하면 좋은 코드가 나올 것 같아요.

```java
class Calculator {
    public int calculate(String text) {
        String[] split = splitText(text);
        int[] numbers = stringArrToIntArr(split);
        return calculate(numbers)
    }
    
    private String[] splitText(String text) { ... } // private 제어

    private int[] stringArrToIntArr(String[] textSplitArr) { ... } // private 제어
}
```

`splitText`, `stringArrToIntArr`을 노출시키지 않고
calculate 행위만으로 Calculator가 내부적으로 처리할 수 있게 만드는 것이 좋을 것 같아요